test/unit/control-bar/text-track-controls/text-track-menu-item.test.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-env qunit */
+import TextTrackMenuItem from '../../../../src/js/control-bar/text-track-controls/text-track-menu-item.js';
+import TestHelpers from '../../test-helpers.js';
+import sinon from 'sinon';
+
+QUnit.module('TextTrackMenuItem', {
+  beforeEach(assert) {
+    this.clock = sinon.useFakeTimers();
+    this.player = TestHelpers.makePlayer();
+  },
+  afterEach(assert) {
+    this.player.dispose();
+    this.clock.restore();
+  }
+});
+
+QUnit.test('clicking should enable the selected track', function(assert) {
+  assert.expect(2);
+
+  const foo = this.player.addTextTrack('captions', 'foo', 'en');
+
+  const fooItem = new TextTrackMenuItem(this.player, {
```

Comment:
this isn't cleaned up/disposed.

Reply:
`fooItem` and `barItem` below aren't cleaned up either.

Reply:
this object is never added to a menu/the player, so it won't get disposed automatically. It's probably not a huge deal but best to clean up as much as we can to reduce any potential memory leak issues with tests.

Reply:
Right, right.